examples/ChatQnA/deploy/monitoring.md

```diff
@@ -0,0 +1,133 @@
+
+# Monitoring ChatQnA deployment
+
+Now that you have deployed the ChatQnA example, let's talk about monitoring the performance of the microservices in the ChatQnA pipeline.
+
+Monitoring the performance of microservices is crucial for ensuring the smooth operation of the generative AI systems. By monitoring metrics such as latency and throughput, you can identify bottlenecks, detect anomalies, and optimize the performance of individual microservices. This allows us to proactively address any issues and ensure that the ChatQnA pipeline is running efficiently.
+
+This document will help you understand how to monitor in real time the latency, throughput, and other metrics of different microservices. You will use **Prometheus** and **Grafana**, both open-source toolkits, to collect metrics and visualize them in a dashboard.
+
+### Set up the Prometheus server
+
+Prometheus is a tool used for recording real-time metrics and is specifically designed for monitoring microservices and alerting based on their metrics.
+
+The `/metrics` endpoint on the port running each microservice exposes the metrics in the Prometheus format. The Prometheus server scrapes these metrics and stores them in its time series database. For example, metrics for the Text Generation Interface (TGI) service are available at:
+```bash
+http://${host_ip}:9009/metrics 
+```
+
+To set up the Prometheus server, follow these steps:
+
+#### 1. Download Prometheus:
```

Comment:
... and all these numbered titles should be one level up (### not ####).  Also all these headings shouldn't end with `:`